packages/main/src/Avatar.ts

```diff
@@ -322,8 +294,7 @@ class Avatar extends UI5Element implements ITabbable {
 	/**
 	 * Returns the effective avatar size.
 	 * @readonly
```

Comment:
remove readonly tag. getter are readonly by default. Same for _effectiveBackgroundColor